docker/Dockerfile.debian

```
@@ -0,0 +1,32 @@
+FROM golang AS builder
+
+RUN mkdir /src
+ADD . /src
+
+RUN cd /src && \
+	mkdir -p output/bin && \
+	make V=1 bin/step
+
+FROM debian:bullseye
+
+ENV STEP="/home/step"
+ENV STEPPATH="/home/step"
+ARG STEPUID=1000
+ARG STEPGID=1000
+
+RUN apt-get update \
+        && apt-get upgrade \
+        && apt-get install -y --no-install-recommends curl \
+        && addgroup --gid ${STEPGID} step \
+        && adduser --disabled-password --uid ${STEPUID} --gid ${STEPGID} step \
+        && chown step:step /home/step
```

`chown` is not needed, adduser will do that by default:

```suggestion
        && adduser --disabled-password --uid ${STEPUID} --gid ${STEPGID} step
```